### ISSUE TITLE
Problem: slug field in Format and FormatVersion is not ensuring uniqueness

### ISSUE THREAD

**Issue by sevein** [artefactual/archivematica-fpr-admin#64]
While working on #63 we found that we had two formats and their corresponding formatversions that had the very same description. The latest one added took the same slug causing MultipleObjectsReturned exceptions.

I think that the slug fields for these two models should be updated so they ensure uniqueness. This is something that the old `django-slugfield` seems to support out of the box. Perhaps we can also move to Django's SlugField?

**Comment by sevein** [comment]
How to reproduce:
1. Go to formats
2. Add a new format
3. Use description: `ADX Audio Format`

You will be redirected to the new record. The slug given will clash with the existing `adx-audio-format` causing the following error:

```
MultipleObjectsReturned at /format/adx-audio-format/

get() returned more than one Format -- it returned 2!
```

**Comment by ablwr** [comment]
This seems like it was resolved? I know when testing, slugs will add -1, -2, -3 if something with the same slug already exists.

**Comment by sevein** [comment]
I think so, sounds like #70 fixed it! It's funny, I look at #70 and I can barely remember I did that... :thinking:
